src/test/java/org/zalando/nakadi/service/EventStreamTest.java

```diff
@@ -294,6 +295,11 @@ private static NakadiKafkaConsumer endlessDummyConsumer() throws NakadiException
 
     private static String jsonBatch(final String partition, final String offset,
                                     final Optional<List<String>> eventsOrNone) {
+        return jsonBatch(partition, offset, eventsOrNone, null);
+    }
+
+    private static String jsonBatch(final String partition, final String offset,
+                                    final Optional<List<String>> eventsOrNone, final String metadata) {
```

Comment:
If you expect that metadata can be null - the parameter should be @Nullable

However for me it makes more sense to make it Optional